src/app/machines/views/MachineDetails/MachineInstances/MachineInstances.test.tsx

```diff
@@ -162,32 +121,20 @@ describe("MachineInstances", () => {
         ],
       }),
     ];
-    const store = mockStore(state);
 
-    const wrapper = mount(
-      <Provider store={store}>
-        <MemoryRouter
-          initialEntries={[
-            { pathname: "/machine/abc123/instances", key: "testKey" },
-          ]}
-        >
-          <CompatRouter>
-            <Routes>
-              <Route
-                element={<MachineInstances />}
-                path="/machine/:id/instances"
-              />
-            </Routes>
-          </CompatRouter>
-        </MemoryRouter>
-      </Provider>
+    renderWithBrowserRouter(
+      <Routes>
+        <Route element={<MachineInstances />} path="/machine/:id/instances" />
+      </Routes>,
+      {
+        state,
+        route: "/machine/abc123/instances",
+      }
```

Comment:
You can pass a `routePattern` to `renderWithBrowserRouter`:
```suggestion
    renderWithBrowserRouter(
      <MachineInstances />,
      {
        state,
        route: "/machine/abc123/instances",
        routePattern: "/machine/:id/instances",
      }
```

Reply:
Please also update other instances of this in this PR @Jay-Topher